core/dbt/events/stubs.py

```diff
@@ -0,0 +1,44 @@
+from typing import (
```

Comment:
is the plan for this file to grow a lot for now and then we can delete it when we turn on mypy everywhere?